### DISCUSSION TITLE
Command Failed

### DISCUSSION THREAD

**Discussion by wanglu4042** [Xilinx/finn#826] [Q&A] [answered]
Hi, I encountered an error at Hardware Generation when running the example cnv_end2end_example given by FINN
The error is :
![image](https://github.com/Xilinx/finn/assets/132076700/456579a4-8a95-4c7f-89cf-27c8680f615a)
And the exception below asked me to check the log:
![image](https://github.com/Xilinx/finn/assets/132076700/419284bd-ff89-44fe-88d3-05cd4e508c6e)
And the log is here:
![image](https://github.com/Xilinx/finn/assets/132076700/a6098904-db75-4633-b42a-d89d764573ad)
How can I fix this problem? I had the same problem when I implemented my own work, so I ran example to check if my work has a problem, but there is a same error in running the example.
Could you help me with this error?
Thanks.

**Comment by fpjentzsch** [comment]
Hi, you'll need to theck the logs for the specific failed run that is mentioned ("top_StreamingDataflowPartition_1_0_synth_1") in the *.runs directory of the project.

**Reply by wanglu4042** [reply]
Hi, dear Mr.fpjentzsch, thanks for your reply, I find a file called 'finn_zynq_link.runs', and this is what it looks like
![image](https://github.com/Xilinx/finn/assets/132076700/5e6b128e-e2d1-4092-adc1-7e83011aff31)
and in the file 'top_StreamingDataflowPartition_1_0_synth_1', ther is a 'runme.log' and shows that 
![image](https://github.com/Xilinx/finn/assets/132076700/88fdfc77-77e0-4033-946f-992be0aca3db)
But there is not 'runme.log' in file 'synth_1' and 'impl_1'.
What caused my error ? How to fix them?
Looking forwards to your reply.
Thanks again

**Reply by fpjentzsch** [reply, marked as answer]
This could indicate that Vivado is running out of memory and crashing. Could you try to observe the RAM utilization during synthesis or lower the number of worker threads?

**Reply by wanglu4042** [reply]
Thanks for your reply, Mr.fpjentzsch.
I solved this problem, as you said, the reason of this error is that the memory I gave to my ubuntu is not enough, I tried 8G memory, and this error disappeared.
Thanks to your reply!
